changelog.d/756.bugfix

```diff
@@ -0,0 +1 @@
+Fix issue on push notification where devices where not emitting sound/vibrating
```

Comment:
this changelog name should be 736.bugfix to refer to the exisitng issue https://github.com/tchapgouv/tchap-android/issues/736